CDCgov/prime-reportstream
# Experience/7280 removed out dated CSV related resources

Removed out dated CSV related resources

Test Steps:
1. Go to [resources](https://reportstream.cdc.gov/resources)
2. Do not see the CSV schema documentation guide Card and the CSV upload guide card

## Changes
![Screenshot 2022-11-23 at 10 25 40 AM](https://user-images.githubusercontent.com/102491809/203621256-90c08995-1267-476b-ae99-df9faf377a4c.png)

## Checklist

### Testing
